src/domain.cc

```diff
@@ -107,6 +109,14 @@ void Domain::Initialize(Handle<Object> exports)
   NODE_DEFINE_CONSTANT(exports, VIR_DOMAIN_DEVICE_MODIFY_LIVE);
   NODE_DEFINE_CONSTANT(exports, VIR_DOMAIN_DEVICE_MODIFY_CONFIG);
 
+  //virDomainMetadataType
+  NODE_DEFINE_CONSTANT(exports, VIR_DOMAIN_METADATA_DESCRIPTION);
+  NODE_DEFINE_CONSTANT(exports, VIR_DOMAIN_METADATA_TITLE);
+  NODE_DEFINE_CONSTANT(exports, VIR_DOMAIN_METADATA_ELEMENT);
+#ifdef VIR_DOMAIN_METADATA_LAST
```

Comment:
It's worse than that.  I guess metadata access is relatively new.  The CI checks both completely bomb.  

And yes, those #ifdef are there because they are in the libvirt docs but not in my system.

I'm going to have to figure out much more #ifdef'ing or something...